cmd/estuary-shuttle/main.go

```diff
@@ -1373,6 +1373,7 @@ func (s *Shuttle) addrsForShuttle() []string {
 }
 
 func (s *Shuttle) createContent(ctx context.Context, u *User, root cid.Cid, fname string, cic util.ContentInCollection) (uint, error) {
+	log.Debugf("createContent> cid: %v, filename: %s, collection: %+v", root, fname, cic)
```

Comment:
you forgot this

Reply:
I was intending on letting it there on purpose for instrumentation's sake. Do you want me to use another function or some other format?